test/postgres.test.js

```diff
@@ -448,3 +449,108 @@ describe('postgres store API V2.0.0', function () {
     })
   })
 })
+
+var siDefault = Seneca({
+  default_plugins: {
+    'mem-store': false
+  }
+})
+
+var siCustom = Seneca({
+  default_plugins: {
+    'mem-store': false
+  }
+})
+
+describe('Column Names conversions', function () {
+  describe('Default CamelCase to snake_case conversion', function () {
+    before({}, function (done) {
+      siDefault.use(require('..'), DefaultConfig)
+      siDefault.ready(function () {
+        siDefault.use(require('seneca-store-query'))
+        siDefault.ready(done)
+      })
+    })
+
+    beforeEach(clearDb(siDefault))
+    beforeEach(createEntities(siDefault, 'foo', [{
+      fooBar: 'fooBar',
+      bar_foo: 'bar_foo'
+    }]))
+
+    it('should not alter CamelCase column names', function (done) {
+      var foo = siDefault.make('foo')
+
+      foo.list$({native$: 'SELECT * FROM foo where "fooBar" = \'fooBar\''}, function (err, res) {
```

Comment:
actually this is one of the tested points that the insert works ok. Previously by default it was trying to insert into foo_bar not fooBar.  As long as there is one result returned for sure it contains the inserted data. I'll data the extra checks as an additional protection